xml/System.Web.Script.Serialization/JavaScriptSerializer.xml

```diff
@@ -16,14 +16,19 @@
   <Interfaces />
   <Docs>
     <summary>
-      <see href="https://www.newtonsoft.com/json">Json.NET</see> should be used for serialization and deserialization. Provides serialization and deserialization functionality for AJAX-enabled applications.</summary>
+      Provides serialization and deserialization functionality for AJAX-enabled applications.
+    </summary>
     <remarks>
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Web.Script.Serialization.JavaScriptSerializer> class is used internally by the asynchronous communication layer to serialize and deserialize the data that is passed between the browser and the Web server. You cannot access that instance of the serializer. However, this class exposes a public API. Therefore, you can use the class when you want to work with JavaScript Object Notation (JSON) in managed code.  
+
+> [!IMPORTANT]
```

Comment:
Can you please share the internal docs link so I can take a quick glance at how this will end up looking like on the docs page.

Reply:
@ahsonkhan When the CI finishes, click on "Details", then scroll down to the preview link for this API.

Reply:
I don't see it:
https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C25%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C202003250031355427-3758%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=NFFwGrzp7innzIOjl3ukNRilz7HD1L0hOknTawYzb98%3D&st=2020-03-25T01%3A43%3A15Z&se=2020-04-25T01%3A48%3A15Z&sp=r

![image](https://user-images.githubusercontent.com/6527137/77493997-cf4a5800-6e01-11ea-9f74-2c36752a0790.png)

Reply:
OK, I got it. I took one of the other links and just navigated to the `JavaScriptSerializer` type:
https://review.docs.microsoft.com/en-us/dotnet/api/system.web.script.serialization.javascriptserializer?view=netframework-4.8&branch=pr-en-us-3758

Reply:
Why does it say that this page isn't available for netfx 4.8?
https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json?view=netcore-3.1&branch=pr-en-us-3758&viewFallbackFrom=netframework-4.8
![image](https://user-images.githubusercontent.com/6527137/77494171-497adc80-6e02-11ea-9a17-7ee222e91778.png)

That could confuse users when we just said that for netfx 4.7.2+ use `S.T.J`. Is the platform support matrix in the docs incomplete? I would have expected anything that is netstanard2.0 compatible to have an entry in the appropriate netfx versions of the page (rather than redirecting to the .NET core version).

Reply:
> I don't see it
That's odd. But I'm glad you found it by navigating into another API.
> Why does it say that this page isn't available for netfx 4.8?
It's also not showing up [in NetStandard2.0](https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json?view=netcore-3.1&branch=pr-en-us-3758&viewFallbackFrom=netstandard-2.0). Do you see the same problem in the production MS Docs site?

Reply:
> Do you see the same problem in the production MS Docs site?

Yes.

Reply:
@joelmartinez can you help us figure out why System.Text.Json is not showing up in the expected monikers?

Reply:
I think @mimisasouvanh might be better point of contact here.